shadowreader/tests/test_app.py

```diff
@@ -50,7 +50,7 @@ def test_app_init_w_identifier():
 
 def test_app_str():
     mytime = MyTime(epoch=1522091259)
-    base_url = 'http://qa-21-www.pytest.com'
+    base_url = 'http://www.shadowreader-unit-test.com'
```

Comment:
For the test, use a domain that will _never_ be valid, e.g. "shadowreader.example.com".